client/chainlink_models.go

```diff
@@ -256,18 +261,16 @@ observationSource = """
 
 // PipelineSpec common API call pipeline
 type PipelineSpec struct {
-	URL         string
-	Method      string
-	RequestData string
-	DataPath    string
+	BridgeTypeAttributes BridgeTypeAttributes
+	DataPath             string
 }
 
 func (d *PipelineSpec) Type() string {
 	return "common_pipeline"
 }
 
 func (d *PipelineSpec) String() (string, error) {
-	sourceString := `fetch    [type=http method={{.Method}} url="{{.URL}}" requestData="{{.RequestData}}"];
+	sourceString := d.BridgeTypeAttributes.Task() + `
```

Comment:
Use `fmt.Sprintf` for concatenation as much as possible.

Reply:
Can't we use templating here?

Reply:
I will look into using templating. Does templating support something like {{.BridgeTypeAttributes.RequestData}} where you need to go multiple levels down?

Reply:
Looks like templates do support that, should be an easy update.